rasterio/_filepath.pyx

```diff
@@ -117,7 +117,7 @@ cdef void* filepath_open(void *pUserData, const char *pszFilename, const char *p
     try:
         file_wrapper = filesystem_info[pszFilename]
     except KeyError:
-        log.error("File-like object not found in virtual filesystem: %s", pszFilename)
```

Comment:
Thoughts about changing it to `info` only if  `pszFilename.endswith(('.aux','.AUX')` and `error` otherwise?

Reply:
@snowman2 I hesitate, because 3.7.0 added .hdr and .HDR and other versions might add new ones. It could be whack-a-mole.

Reply:
Another option could be to check if the extension is in [raster_driver_extensions](https://github.com/rasterio/rasterio/blob/6d21d4194bcaf495b13db4498250d10968bd5600/rasterio/drivers.py#L33).